api/mfx_dispatch/linux/mfxloader.cpp

```diff
@@ -52,6 +54,7 @@ namespace MFX {
     #else
         #define LIBMFXSW "libmfxsw64.so.1"
         #define LIBMFXHW "libmfxhw64.so.1"
+        #define ONEVPLRT "libmfx-gen.so.1.2"
```

Comment:
Remove `.2`, we need to load library via soname.

Reply:
Removed